bundler.js

```diff
@@ -137,9 +137,8 @@ export async function installBundler(bundlerVersionInput, rubygemsInputSet, lock
   }
 
   const gem = path.join(rubyPrefix, 'bin', 'gem')
-  // Workaround for https://github.com/rubygems/rubygems/issues/5245
   // and for https://github.com/oracle/truffleruby/issues/2780
-  const force = ((platform.startsWith('windows-') && engine === 'ruby' && floatVersion >= 3.1) || (engine === 'truffleruby')) ? ['--force'] : []
+  const force = engine === 'truffleruby' ? ['--force'] : []
```

Comment:
Could you try removing this line and the comment entirely?
I think at the source it was a RubyGems bug to be too careful about conflicts with `bin/bundle`.
And I guess probably the fix got backported to the recent 3.1 releases.

Reply:
> I think at the source it was a RubyGems bug to be too careful about conflicts with `bin/bundle`.
> And I guess probably the fix got backported to the recent 3.1 releases.

Correction: reading https://github.com/oracle/truffleruby/issues/2780#issuecomment-1308981564 more carefully, the problem happened when updating to latest RubyGems, which added a regression with https://github.com/rubygems/rubygems/pull/2929/files#diff-b25097522a961ed7158012499383b8a8057caf87cbc445d6b168a5b8a27ba45dR224 and that got fixed in newer RubyGems versions.
So we shouldn't need this workaround at all anymore, because anyway it only happened when using latest rubygems and latest rubygems no longer has this issue (I assume).

Reply:
Done! It should be good now if CI is fine.